docs/MUTATION_TESTING.md

```
@@ -0,0 +1,399 @@
+open mutation-test-report/index.html
+```
+
+El reporte muestra:
+
+- **Per-file mutation scores**: Score por cada archivo
+- **Surviving mutants highlighted in red**: Click para ver qué mutación sobrevivió
+- **Quality ratings**: Escala A-E
+
+#### Ejemplo de output
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)